app/dashboard/static/js/app/view-socs-soc-job-kernel.2017.4.js

```diff
@@ -402,19 +61,15 @@ define([
             gitCommit,
             gitURL,
             gitURLs,
-            results,
             tooltipNode;
 
-        results = response.result;
-        if (results.length === 0) {
+        if (response.length === 0) {
```

Comment:
This is still needed?  The function that calls `updateDetails` is already checking for the length of results and this variable is in fact `response.results[0]`.